state/alicloud/tablestore/tablestore_test.go

```diff
@@ -63,15 +64,15 @@ func TestReadAndWrite(t *testing.T) {
 			Value: "value of key",
 			ETag:  ptr.String("the etag"),
 		}
-		err := store.Set(setReq)
+		err := store.Set(context.TODO(), setReq)
```

Comment:
In all tests, I'd use `context.Background()` as not setting a context is intentional.

The idea of `context.TODO()` is that eventually needs to be replaced (and when you perform a search for `TODO` in a repo, they appear).